frontend/src/components/experimentRuns/ExperimentRunsDetachedFilters.ts

```diff
@@ -0,0 +1,33 @@
+import { FilterDescription } from "../../models/paged_items";
+import { FILTER_TYPE } from '../../constants'
+import { capitalize } from "../../utils/capitalize";
```

Comment:
i like how the code looks, the only error i got was this capitalize function/file not appearing. the fix is easy as well.

i only removed the function and added flag.toUpperCase() vs capitalize(flag)

Reply:
You can remove the capitalize file then if there is some built-in function that does that ...